src/utils/seo.ts

```diff
@@ -0,0 +1,19 @@
+export interface Seo {
+	title: string;
+	description: string;
+	og: string;
+}
+
+export const DEFAULT_SEO = {
+	title: 'Parliament Watch - ขับเคลื่อนประชาธิปไตย ร่วมเฝ้าดูความเคลื่อนไหวรัฐสภา',
+	description: 'ติดตามกฎหมาย นโยบาย และการทำงานของนักการเมือง',
+	og: 'https://parliamentwatch.wevis.info/images/og.png'
+};
+
+export const createSeo = (data: Partial<Seo>): Seo => {
+	return {
+		title: data.title ?? DEFAULT_SEO.title,
```

Comment:
Since we postfix every page title with `- Parliament Watch`, to avoid typing that every time, I think we can replace it with another prop like `pageName`
- Title = `pageName - Parliament Watch`
- If  `pageName` is not given, use the default value.

The rest is looking good to me.